clients/admin-ui/src/features/subject-request/EventsAndLogs.tsx

```diff
@@ -0,0 +1,32 @@
+import { Divider, Flex, Heading } from '@fidesui/react';
+import React, { useState } from 'react';
+
+import { ExecutionLog, PrivacyRequest } from '../privacy-requests/types';
+import ActivityTimeline from './ActivityTimeline';
+import EventDetails from './EventDetails';
+
+type EventsAndLogsProps = {
+  subjectRequest: PrivacyRequest;
+};
+
+const EventsAndLogs = ({ subjectRequest }: EventsAndLogsProps) => {
```

Comment:
This component is mostly finished. It's waiting on some mockups to be finished. I talked with @seanpreston and we decided to leave it in the repo without being used because the work was already done here.